.github/workflows/build-deploy.yml

```diff
@@ -40,6 +40,12 @@ jobs:
           name: production-files
           path: ./build
 
+      - name: Run ESLint
+        run: yarn run lint . --ext .js,.jsx,.ts,.tsx
```

Comment:
@Ashara-kosi why not run simply `yarn run lint` ? That should already configured to run what is needed, and if something is not configured properly there, then it should be reconfigured through `package.json`, instead of running a custom command here.

Reply:
@lacabra I have made the edits based on your comments

Thank you for the feedback!